internal/controller/server/repository/job/job.go

```diff
@@ -21,13 +21,11 @@ import (
 
 // Repository interface for job repository functions
 type Repository interface {
-	GetValue(ctx context.Context, jobName string) (*protofiles.Metadata, error)
-	CheckJobIsAvailable(ctx context.Context, jobName string) (bool, error)
+	GetMetadata(ctx context.Context, jobName string) (*protofiles.Metadata, error)
 	Save(ctx context.Context, jobID uint64, executionData *protofiles.RequestToExecute) error
```

Comment:
I guess it would be better if we provide more context here into what we are saving and add a suffix to Save